### ISSUE TITLE
Possible to rename folder and files?

### ISSUE THREAD

**Issue by TechieCrow** [webbertakken/unity-actions#63]
Is it possible to rename the "StandaloneWindows64" folder and files to the project name before the artifact step somehow?

I've got the basics working with no problems but it would be nice to eventually push the finished build to a download server but would need it to use the project name instead of the platform name. :)

**Comment by DanielParra159** [comment]
As I can see here [action.yml](https://github.com/webbertakken/unity-builder/blob/master/action.yml), maybe you can use `buildName` param in your build action.

**Comment by TechieCrow** [comment]
(not to help, don't open a new issue before coffee)

Thanks! I actually feel a little dumb now haha.

**Comment by webbertakken** [comment]
You can refer to the readme for an explanation of all parameters. 

For builder see: https://github.com/webbertakken/unity-builder#configuration-options
